app/javascript/controllers/planner_controller.js

```diff
@@ -0,0 +1,50 @@
+import { Controller } from "@hotwired/stimulus"
+
+export default class extends Controller {
```

Comment:
I broke this out from the `count_controller.js`... because there was just too much custom stuff going on here.

This handles both updating the "create N drafts" button text, and enabling/disabling it based on if you've selected at least 1 date and a num_ad_breaks.